### ISSUE TITLE
Initfence should be started earlier

### ISSUE THREAD

**Issue by OnGle** [turnkeylinux/tracker#984]
The initfence starts after services including apache, mysql and sshd leaving a small window where the underlying webserver is exposed to the network and where if a user were to ssh in, they would not be prompted with the inithooks. Under normal circumstances this shouldn't be a problem due to the passwords being randomly generated before the inithooks get run interactively and the fact that often computing platforms have a few minutes before networking becomes available to the appliance.

However under exceptional circumstances for example a self-hosted server running mibew (that still has https://github.com/turnkeylinux/tracker/issues/941 unpatched) this could be a real issue.

**Comment by JedMeister** [comment]
Good call. I guess the answer is to move the initfence higher in the priority order on headless appliances?!

**Comment by JedMeister** [comment]
I was hoping to make that initfence start earlier by moving the [`29preseed`](https://github.com/turnkeylinux/buildtasks/blob/master/patches/headless/overlay/usr/lib/inithooks/firstboot.d/29preseed) inithook earlier (I moved it to `11preseed`) but there is obviously something I'm missing/ As it meant that preseeded Hub servers still had the initfence enabled and required interactive running of firstboot scripts.

The plan is still good, but we need to reconsider how we achieve this. Perhaps we need to enable the initfence by default on all headless builds and disable it when preseeded?!

**Comment by JedMeister** [comment]
Now that the init-fence is provided via a systemd service, another way to do this would be enable the service at build time for the relevant buildtypes?! Regardless, I'm moving to v17.0 for now...
